apps/src/templates/teacherDashboard/CourseOfferingHelpers.jsx

```diff
@@ -72,6 +73,7 @@ export const subjectsAndTopicsOrder = [
   'science',
   'english_language_arts',
   'history',
+  'digital_literacy',
```

Comment:
Thanks for adding this here! I'm hoping the weirdly written tests caught this too 😅